Core/Model.py

```diff
@@ -296,3 +295,77 @@ def generate_direct_transition_system(self, max_time: float = np.inf, max_size:
             time.sleep(1)
 
         return ts
+    
+    def export_sbml(self) -> libsbml.SBMLDocument:
+        """
+        Convert model to a SBML model using SBML-multi package (libsbml).
+        :return: SBML document
+        """
+        test_model = ModelSBML()
+        test_model.docPlug.setRequired(True)
+        unique_complexes, unique_params = self.create_unique_complexes_and_params()
+
+        test_model.create_basic_species_types(self.atomic_signature, self.structure_signature) #1
+        test_model.create_all_species_compartments_and_complex_species_types(unique_complexes)#2
+        test_model.create_all_reactions(self.rules)#3
+        test_model.create_reaction_for_isomorphisms(unique_complexes)#3.1
+        test_model.create_parameters(self.definitions, unique_params)#4
+        test_model.set_initial_amounts(self.init)#5
+
+        return test_model.document
+
+    def create_unique_complexes_and_params(self):
+        """
+        Extracts unique complexes and compartments from rules
+        splits complexes by its isomorphisms
+        :return: dict of unique complexes mapped to list of its isomorphisms and set of unique params
+        """
+        #here all complexes will be (complexAgent:{(complexAgent, SBML_code_isomprphism1),(complexAgent, SBML_code_isomprphism2)... })
```

Comment:
Yes, thats what i thought at first. The reason why i keep remembering both is this:

Saving complex agent of different isomorphism is useful so we can work with it.. it can have different ordering of inner agents and that information could be lost otherwise.

By saying that we could just remember complex_agent and forget about SBML_code_isomorphism because it can be generated from it. Yes that is true, however putting agents into dict hapens only if they are different by eq function an that function does not take into account possible difference of ordering